server/views/pages/prisonerSummary/prisonerSummary.njk

```diff
@@ -0,0 +1,110 @@
+{% extends "partials/layout.njk" %}
+
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "macros/renderFields.njk" import renderFields %}
+
+{% set pageTitle = applicationName + " - Report" %}
+{% set bodyClasses = "app-view-report" %}
+
+{% block beforeContent %}
+{{ super() }}
+
+{{ govukBackLink({
+    text: "Back",
+    href: backLink
+  }) }}
+{% endblock %}
+
+{% block content %}
+
+<h1 class="govuk-heading-l">Prisoners involved</h1>
+
+{% if showTable %}
+
+  {% set rows = [] %}
+  {% for prisoner in report.prisonersInvolved %}
+    {% if report.createdInNomis %}
+      {% set rows = (rows.push(
+        [
+          { text: prisoner.prisonerNumber + ': ' + (prisoner | nameOfPerson) },
+          { text: prisonerInvolvementLookup[prisoner.prisonerRole] or prisoner.prisonerRole },
+          { text: prisonerOutcomeLookup[prisoner.outcome] or prisoner.outcome },
+          { text: prisoner.comment },
+          { html: "<a href='/reports/" + report.id + "'>Remove</a>" + ' | ' + "<a href='/reports/'>Edit</a>" }
+        ]
+        ), rows) %}
+    {% else %}
+      {% set rows = (rows.push(
+        [
+          { text: prisoner.prisonerNumber + ': ' + (prisoner | nameOfPerson) },
+          { text: prisonerInvolvementLookup[prisoner.prisonerRole] or prisoner.prisonerRole },
+          { text: prisoner.comment },
+          { html: "<a href='/reports/" + report.id + "'>Remove</a>" + ' | ' + "<a href='/reports/'>Edit</a>" }
+        ]
+        ), rows) %}
+    {% endif %}
+  {% endfor %}
+
+  {{ govukTable({
+    head: tableHeading,
```

Comment:
i would add a class, one that will be shared with the staff involvement table, like `app-involvement-table` or similar. because looking at the designs, we will need to add a style to right-alight the last table header cell:
```scss
.app-involvement-table {
  th:last-child {
    text-align: right;
  }
}
```